performance-metrics/src/performance_tests.rs

```diff
@@ -634,7 +656,7 @@ pub fn performance_block_io(control: &PerformanceTestControl) -> f64 {
     let mut child = GuestCommand::new(&guest)
         .args(&["--cpus", &format!("boot={}", num_queues)])
         .args(&["--memory", "size=4G"])
-        .args(&["--kernel", direct_kernel_boot_path().to_str().unwrap()])
+        .args(&["--kernel", fio_test_kernel_boot_path().to_str().unwrap()])
```

Comment:
I would suggest to change the test itself to use cold plug for the test image. This way wouldn't make any functional difference for the performance test, but will avoid few tweaks needed for aarch64: 1) the need for UEFI firmware; 2) the longer boot-time when using UEFI firmware boot.

Reply:
Yeah, cold-plug is better.